examples/rust/custom_store_subscriber/src/main.rs

```diff
@@ -202,8 +205,7 @@ impl ChunkStoreSubscriber for TimeRangesPerEntity {
                         .last_key_value()
                         .map_or(TimeInt::MAX, |(time, _)| *time),
                 );
-                let time_range = timeline.format_time_range_utc(&time_range);
-                println!("  {time_range}");
+                println!("  {timeline}: {time_range:?}");
```

Comment:
It would actually be nice to show how to get the type of the timeline and how to use it to print it in a readable way here.